html-css/examples/chapter10.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- USING EXTERNAL CSS -->
+    <link rel="stylesheet" href="css/chapter10.css">
+    <!-- USING INTERNAL CSS -->
```

Comment:
Pls not write uppercase if it is not very important content.

`<!-- Using external CSS-->` is good enough

Reply:
Thank you for your comments. I will learn from experience